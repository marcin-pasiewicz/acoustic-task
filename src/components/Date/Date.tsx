import React, { ReactElement } from 'react'
import moment from 'moment'
import {FULL_DATE_WITH_SECONDS} from './../../constants/dateFormats'

interface Props {
  value: string;
}

export default function Date({value}: Props): ReactElement {
  return (
    <div>
      {moment(value).format(FULL_DATE_WITH_SECONDS)}
    </div>
  )
}
