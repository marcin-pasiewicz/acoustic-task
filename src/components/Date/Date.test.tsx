import React from "react";
import { render } from "@testing-library/react";
import Date from "./Date";

describe("<Date/>", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Date value="2019-12-13T17:00:00Z" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
