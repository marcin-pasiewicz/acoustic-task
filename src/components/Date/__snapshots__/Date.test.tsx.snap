// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Date/> renders correctly 1`] = `
<DocumentFragment>
  <div>
    December 13th 2019, 6:00:00 pm
  </div>
</DocumentFragment>
`;
